src/pages/core/external-plugins/app-data.md

```diff
@@ -0,0 +1,248 @@
+| ------------- | --------------------------- |
+| dataAuthority | PluginAuthority             |
+| schema        | ExternalPluginAdapterSchema |
+
+### dataAuthority
+
+{% dialect-switcher title="AttributeList" %}
+{% dialect title="JavaScript" id="js" %}
+
+```ts
+const dataAuthority = {
+  type: 'Address',
+  address: publicKey('11111111111111111111111111111111'),
+}
+```
+
+{% /dialect %}
+
+{% dialect title="Rust" id="rust" %}
+
+```rust
+use mpl_core::types::{PluginAuthority}
+
+let data_authority = Some(PluginAuthority::Address {address: authority.key()}),
+```
+
+{% /dialect %}
+{% /dialect-switcher %}
+
+### schema
+
+The schema determines the type of data that is being stored within the AppData plugin. Certain schemas will be DAS compatible and indexed by DAS RPC providers. These schema data types include:
+
+| Arg               | DAS Supported |
+| ----------------- | ------------- |
+| Binary (Raw Data) | ✅            |
+| Json              | ✅            |
+| MsgPack           | ✅            |
+
+{% dialect-switcher title="Writing data to the AppData plugin" %}
+{% dialect title="JavaScript" id="js" %}
+
+```ts
+import { ExternalPluginAdapterSchema } from '@metaplex-foundation/mpl-core'
+
+// Chose from Binary, Json or MsgPack
+const schema = ExternalPluginAdapterSchema.Json
+```
+
+{% /dialect %}
+
+{% dialect title="Rust" id="rust" %}
+
+```rust
+// Chose from Binary, Json or MsgPack
+let schema = ExternalPluginAdapterSchema::Json
+
+```
+
+{% /dialect %}
+
+{% /dialect-switcher %}
+
+## Adding the AppData Plugin to an Asset
+
+{% dialect-switcher title="Adding a Attribute Plugin to an MPL Core Asset" %}
+{% dialect title="JavaScript" id="js" %}
+
+```ts
+import { publicKey } from '@metaplex-foundation/umi'
+import { addPlugin, ExternalPluginAdapterSchema } from '@metaplex-foundation/mpl-core'
+
+const assetSigner = generateSigner(umi);
+const dataAuthority = publicKey('11111111111111111111111111111111')
+
+await create(umi, {
+  asset: asset.publicKey,
+  name: "My Asset",
+  uri: "https://example.com/my-assets.json"
+  plugins: [
+        {
+            type: 'AppData',
+            dataAuthority,
+            schema: ExternalPluginAdapterSchema.Json,
+        },
+    ],
+}).sendAndConfirm(umi)
+
+// Alternatively you could add the plugin to an existing Asset
+
+await addPlugin(umi, {
+  asset,
+  plugin: {
+        type: 'AppData',
+        dataAuthority,
+        schema: ExternalPluginAdapterSchema.Json,
+    },
+})
+```
+
+{% /dialect %}
+
+{% dialect title="Rust" id="rust" %}
+
+```rust
+use mpl_core::{
+    instructions::AddExternalPluginAdapterV1Builder,
+    types::{Attribute, Attributes, Plugin,
+    ExternalPluginAdapterInitInfo, AppDataInitInfo,
+    PluginAuthority},
+};
+use solana_client::nonblocking::rpc_client;
+use solana_sdk::{pubkey::Pubkey, signature::Keypair, signer::Signer, transaction::Transaction};
+use std::str::FromStr;
+
+pub async fn add_attributes_plugin() {
```

Comment:
I see the code adds app data, but the function is called `add_attributes_plugin()`